_includes/vidioPlayer.html

```diff
@@ -0,0 +1,14 @@
+<!-- Feel free to change the width and height to your desired video size. -->
+
+<div class="embed-container">
+  <iframe
+      class="vidio-embed"
```

Comment:
I would like to keep this include file as barebones as possible without any external CSS or modifications. Could you remove `class="vidio-embed"` and the added javascript script?

Reply:
The player works fine without the added script `<script src="//cdn0-a.production.vidio.static6.com/assets/javascripts/vidio-embed.js"></script>`